boards/b-l072z-lrwan1/Makefile.features

```diff
@@ -3,7 +3,7 @@ FEATURES_PROVIDED += periph_dma
 FEATURES_PROVIDED += periph_i2c
 FEATURES_PROVIDED += periph_rtc
 FEATURES_PROVIDED += periph_rtt
-FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_spi periph_spi_gpio_mode
```

Comment:
Also could you put these in another line.

Reply:
It follows the same scheme used with `periph_gpio periph_gpio_irq` in CPU.

Reply:
Ok.